applications/fishing-map/src/features/timebar/Timebar.tsx

```diff
@@ -258,21 +223,23 @@ const TimebarWrapper = () => {
           <Fragment>
             {timebarVisualisation === TimebarVisualisations.Heatmap && <TimebarActivityGraph />}
             {timebarVisualisation === TimebarVisualisations.Vessel &&
-              (syncedTracks && syncedTracks.length <= MAX_TIMEBAR_VESSELS ? (
+              (tracks && t.length <= MAX_TIMEBAR_VESSELS ? (
```

Comment:
`t` is the function to translate so, I guess you wanted to do:
```suggestion
              (tracks && tracks.length <= MAX_TIMEBAR_VESSELS ? (
```